.githooks/pre-commit

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 echo "Formatting Python code with black"
-black .
+black -l 110 .
```

Comment:
In future if you change formatting I suggest a separate commit for that -- it'd be easier to tease out the important changes from the others.

Reply:
Duly noted!